scripts/master.py

```diff
@@ -42,5 +43,6 @@
         KERAS2_API_MASTER,
         TUNER_MASTER,
         HUB_MASTER,
+        RS_MASTER,
```

Comment:
Isn't that the thing you should remove for now so that KERAS_RS doesn't appear in the menu?

Reply:
Nope, all of that is in the HTML files